fe/fe-core/src/main/java/com/starrocks/catalog/PartitionInfo.java

```diff
@@ -190,6 +191,7 @@ public void dropPartition(long partitionId) {
         idToDataProperty.remove(partitionId);
         idToReplicationNum.remove(partitionId);
         idToInMemory.remove(partitionId);
+        GlobalStateMgr.getCurrentState().getAnalyzeMgr().dropPartition(partitionId);
```

Comment:
Are you 100% sure that "GlobalStateMgr.getCurrentState() is not null" and "GlobalStateMgr.getCurrentState().getAnalyzeMgr() is not null" all the time.

Reply:
If this is executed in a replay thread, the current state may be null.